### PR TITLE
MARMOTTA-428: add OSGi support to ldcache and ldclient libraries

### PR DESCRIPTION
**Pull request by acoburn**
Addresses: https://issues.apache.org/jira/browse/MARMOTTA-428

### CODE REVIEW

File: commons/marmotta-sesame-tools/marmotta-model-vocabs/pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     </parent>
 
     <artifactId>marmotta-model-vocabs</artifactId>
-    <packaging>jar</packaging>
+    <packaging>bundle</packaging>
```
Comment (wikier) [inline commons/marmotta-sesame-tools/marmotta-model-vocabs/pom.xml]: why `marmotta-model-vocabs` here? it has nothing to do with either LDPath or LDCache...

Comment (acoburn) [inline commons/marmotta-sesame-tools/marmotta-model-vocabs/pom.xml]: `marmotta-model-vocabs` is used by both `ldclient-provider-facebook` (https://git.io/vKZb7) and `ldclient-provider-ldap` (https://git.io/vKZNf). A compelling alternative would be to embed that dependencies in the relevant ldclient-provider-* bundles.

**Comment by acoburn** [comment]
I am curious about the status of this PR. Thanks!

**Comment by wikier** [comment]
Awaiting for the feedback from @westei...

**Pull request merged**